pkg/blobinfocache/default.go

```diff
@@ -74,3 +74,12 @@ func DefaultCache(sys *types.SystemContext) types.BlobInfoCache {
 	logrus.Debugf("Using SQLite blob info cache at %s", path)
 	return cache
 }
+
+// Cleanup removes the blob info cache directory.
```

Comment:
This might not do anything for caches which already exist in this process, either because they keep a file descriptor open, or because it’s `pkg/blobinfocache/memory`.

I suppose that’s fine (doing anything else would be too hard and it’s not necessary for the use cases), but it should be documented.